bin/pycbc_make_sky_grid

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+"""For a given external trigger (GRB, FRB, neutrino, etc...), generate a sky grid covering its localization error region.
+
+This sky grid will be used by `pycbc_multi_inspiral` to find multi-detector gravitational wave triggers and calculate the
+coherent SNRs and related statistics. 
+
+The grid is constructed following the method described in Section V of https://arxiv.org/abs/1410.6042"""
+
+import numpy as np
+from numpy import zeros
+import h5py
+import argparse
+from pycbc.detector import Detector
+import itertools
+import pycbc.coordinates
```

Comment:
You can remove the `zeros` and `pycbc.coordinates` imports as they are not used.